.github/workflows/ci.yml

```
@@ -33,10 +33,11 @@ jobs:
   test:
     needs: build
     name: 'Integration Tests'
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.runs_on }}
     timeout-minutes: 8
     strategy:
       matrix:
+        runs_on: [ubuntu-latest, ubuntu-24.04-arm]
         os: [ubuntu, centos, arch]
```

I don't think there will be breaking changes between the amd64 and arm64 version of a distribution. I suggest to disable ArchLinux on ARM64:

```suggestion
        os: [ubuntu, centos, arch]
        exclude:
          - runs_on: ubuntu-24.04-arm
             os: arch
```